Documentation/user-guides/alerting.md

```diff
@@ -91,6 +91,24 @@ spec:
       alertmanagerConfig: example
 ```
 
+## Specify Global Alertmanager Config
```

Comment:
I think that the document already needs a rewrite to separate clearly the configuration of Alertmanager and the AlertmanagerConfig CRD but it can be addressed after this PR merges.